client/src/components/common/Modal.tsx

```diff
@@ -0,0 +1,32 @@
+import { ReactNode } from 'react';
+import styled from 'styled-components';
+
+interface ModalProps {
+  children: ReactNode;
+  close: () => void;
+}
+
+export default function Modal({ children, close }: ModalProps) {
+  const handleClick = (e: any) => {
```

Comment:
이벤트 타입 구체적으로 들어가면 좋을것같아요~

```suggestion
  const handleClick = (e: React.MouseEvent<HTMLDivElement, MouseEvent>) => {
```